player.py

```
@@ -0,0 +1,289 @@
+
+
+class PackagePlayer:
+    """
+    Standalone launcher that mounts a .gamepackage and boots directly into gameplay.
+    Bypasses editor UI entirely — presents splash/dashboard then launches game loop.
+    """
+    
+    def __init__(self, package_path: str):
+        self.package_path = os.path.abspath(package_path)
+        self.rm = ResourceManager()
+        self.screen: Optional[pygame.display.Surface] = None
```

Ruff (0.15.12)

[error] Undefined name `Optional`

(F821)